intrinsic functions updated for cloud9 (#200)
aws-samples/cfn101-workshop: content/basics/templates/intrinsic-functions/index.md

Reviewer: Please add C9 instructions to the challenge part, line 155-156

Author: done

Now it is time to update your stack. Go to the AWS console and update your CloudFormation Stack.

:::::tabs{variant="container"}

::::tab{id="cloud9" label="Cloud9"}
1. In the **Cloud9 terminal** navigate to `code/workspace`:
:::code{language=shell showLineNumbers=false showCopyAction=true}
cd cfn101-workshop/code/workspace
:::

> 1. Use the AWS CLI to update the stack. The required parameter `--template-body` have been pre-filled for you. Use `AmiId` which you have hardcoded in `resources.yaml` file earlier.

Reviewer: ```suggestion
1. Use the AWS CLI to update the stack. The required parameter `--template-body` have been pre-filled for you. Replace the `ParameterValue` **MyAmiId** with  the value you have hardcoded in `resources.yaml` file earlier.
```

Author: done

:::code{language=shell showLineNumbers=false showCopyAction=true}

> aws cloudformation update-stack --stack-name cfn-workshop-s3 --template-body file://intrinsic-functions.yaml --parameters '[{"ParameterKey":"AmiID","ParameterValue":"ami-id"}]'

Reviewer: ```suggestion
aws cloudformation update-stack --stack-name cfn-workshop-ec2 --template-body file://intrinsic-functions.yaml --parameters '[{"ParameterKey":"AmiID","ParameterValue":"MyAmiId"}]'
```

Author: done

:::
1. If the `update-stack` command was successfully sent, CloudFormation will return `StackId`.
:::code{language=shell showLineNumbers=false showCopyAction=true}

> "StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-s3/739fafa0-e4d7-11ed-a000-12d9009553ff"

Reviewer: ```suggestion
"StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-ec2/739fafa0-e4d7-11ed-a000-12d9009553ff"
```
